woocommerce/woocommerce
# Analytics: Fix custom date range calendar styling in WP 6.1

### All Submissions:






### Changes proposed in this Pull Request:



This PR includes the `react-dates` CSS that was previously included via WordPress. It is no longer included by WordPress 6.1+.

This CSS is required to properly style the calendar in the Analytics custom date range dropdown.

<img width="392" alt="Screenshot 2022-11-20 at 10 12 15" src="https://user-images.githubusercontent.com/2098816/202911463-9e5ec0d4-4bcb-44e7-9e34-21a7030b47ad.png">

Closes #35645.






### How to test the changes in this Pull Request:



1. Go to any Analytics page.
2. Click on the "Date range" and click "Custom".
3. Verify that the calendar styling is fixed (see above).



### Other information:




### FOR PR REVIEWER ONLY:
